src/components/TokenPairSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenPairSelectorList } from 'components/TokenPairSelectorList'
+
+import ArrowWhite from 'assets/img/arrow-white.svg'
+
+const TokenPairSelectorStyled = styled.div`
+  display: flex;
+  width: 100%;
+
+  > button {
+    width: 100%;
+    height: var(--height-bar-default);
+    background: var(--color-primary);
+    color: var(--color-text-secondary1);
+    appearance: none;
+    border: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 1.6rem;
+    box-sizing: border-box;
+    border-bottom: 0.1rem solid var(--color-border);
+    outline: 0;
+    cursor: pointer;
+    transition: color 0.3s ease-in-out;
+
+    &:hover {
+      color: var(--color-text-primary);
+    }
+  }
+
+  > button > b {
+    font-size: 1.5rem;
+    letter-spacing: 0.03rem;
+  }
+
+  > button > i {
+    font-size: 1.2rem;
+    font-style: normal;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    align-items: center;
+    font-weight: var(--font-weight-medium);
+
+    &::after {
+      display: block;
+      height: 1.2rem;
+      width: 1.2rem;
+      content: '';
+      background: url(${ArrowWhite}) no-repeat center/contain;
+      transform: rotate(90deg);
+      margin: 0 0 0 0.7rem;
+    }
+  }
+`
+
+export const TokenPairSelector: React.FC = () => (
+  <TokenPairSelectorStyled>
+    <button>
+      <b>ETH/USDC</b>
+      <i>Select Pair</i>
+      <TokenPairSelectorList />
```

Comment:
I think this is a select component. Can we do a generic component for any select?

Reply:
It depends what we define under a `select` component. In my opinion this will be a full blown select component where you can do multiple things (tabs, star token pairs, etc.). In my mind it only makes sense to have a generic `select` component for something we can re-use (like we did with the V1 token selector). Curious on your thoughts.

Reply:
should this not be `react-select` or are we scrapping that?

Reply:
Would our purposed token pair select menu, fit the use of `react-select`? (never used it extensively)

Reply:
Good point :) 

Yes and no.

I think in this case it would be more a button that looks like a select? 

Select pair here:
![image](https://user-images.githubusercontent.com/2352112/98950944-eb6a4900-24f9-11eb-973a-af8602d862a6.png)

Opens this:
![image](https://user-images.githubusercontent.com/2352112/98950958-f02efd00-24f9-11eb-9d05-230e92f9c514.png)


So it might be that is not even a select

Reply:
It rather is:
- On click of the `<button>` element (Select pair) it opens a new component.
- The new component consists of:
  - Tabs
  - Each content of a selected tab contains a scrollable list + inline search function

Reply:
cool, it works. So, in reply to @W3stside we don't need the react-select, cause is a button

Reply:
let's not use `react-select` I remember the pain...

Reply:
Well, we are open for the change. But we need to consider we have a working select with functionality to add tokens and so on. If we can leverage that is cool. If we don't need it, we can can make another proposal. Anyways, for now we don't even have a select I think